Core/Newsletter2Go/Bootstrap.php

```diff
@@ -134,6 +121,14 @@ public function onEnlightControllerFrontStartDispatch()
         $this->Application()->Loader()->registerNamespace('Newsletter2Go\Services', $this->Path() . 'Services/');
     }
 
+  public function onStartDispatch(Enlight_Event_EventArgs $args)
```

Comment:
since it is like this everywhere and it works, I would prefer not to change it.
I see it is used same in shopware as well
https://github.com/shopware/shopware/blob/5.7/engine/Library/Enlight/Event/Handler/Plugin.php#L145

Reply:
actually it works in both ways. we have it with `\` in another place in the code